src/language/CodeInspection.js

```diff
@@ -608,11 +608,19 @@ define(function (require, exports, module) {
             toggleCollapsed(prefs.get(PREF_COLLAPSED), true);
         });
     
-    prefs.definePreference(PREF_ASYNC_TIMEOUT, "number", 10000);
+    prefs.definePreference(PREF_ASYNC_TIMEOUT, "number", 10000, {
+        description: Strings.DESCRIPTION_ASYNC_TIMEOUT
+    });
 
-    prefs.definePreference(PREF_PREFER_PROVIDERS, "array", []);
+    prefs.definePreference(PREF_PREFER_PROVIDERS, "array", [], {
+        description: Strings.DESCRIPTION_LINTING_PREFER,
+        values: ["JSLint", "JSHint"],
```

Comment:
If you could just go ahead and remove this line (and also squash commits if you like), I could merge this PR in.